test/specs/reference/tests/01.reference.js

```diff
@@ -6,6 +6,7 @@ exports.execute = function (options) {
             tableName = "reference_table",
             tableNameWithSlash = "table_w_slash",
             tableNameWithCompKey = "table_w_only_composite_key",
+            systemColumnTable = "table_with_system_columns",
```

Comment:
This is not needed since you added your test cases in another file.

Reply:
Removed